plugins/inputs/ravendb/ravendb.go

```diff
@@ -378,52 +375,59 @@ func gatherServer(r *RavenDB, acc telegraf.Accumulator) {
-		"license_utilized_cpu_cores":                        serverResponse.License.UtilizedCpuCores,
-		"license_max_cores":                                 serverResponse.License.MaxCores,
-		"network_tcp_active_connections":                    serverResponse.Network.TcpActiveConnections,
-		"network_concurrent_requests_count":                 serverResponse.Network.ConcurrentRequestsCount,
-		"network_total_requests":                            serverResponse.Network.TotalRequests,
-		"network_requests_per_sec":                          serverResponse.Network.RequestsPerSec,
-		"cluster_node_state":                                mapNodeState(serverResponse.Cluster.NodeState),
-		"cluster_current_term":                              serverResponse.Cluster.CurrentTerm,
-		"cluster_index":                                     serverResponse.Cluster.Index,
-		"databases_total_count":                             serverResponse.Databases.TotalCount,
-		"databases_loaded_count":                            serverResponse.Databases.LoadedCount,
+		"server_version":                                                serverResponse.ServerVersion,
+		"server_full_version":                                           serverResponse.ServerFullVersion,
+		"uptime_in_sec":                                                 serverResponse.UpTimeInSec,
+		"server_process_id":                                             serverResponse.ServerProcessId,
+		"config_server_urls":                                            strings.Join(serverResponse.Config.ServerUrls, ";"),
+		"backup_max_number_of_concurrent_backups":                       serverResponse.Backup.MaxNumberOfConcurrentBackups,
+		"backup_current_number_of_running_backups":                      serverResponse.Backup.CurrentNumberOfRunningBackups,
+		"cpu_process_usage":                                             serverResponse.Cpu.ProcessUsage,
+		"cpu_machine_usage":                                             serverResponse.Cpu.MachineUsage,
+		"cpu_processor_count":                                           serverResponse.Cpu.ProcessorCount,
+		"cpu_assigned_processor_count":                                  serverResponse.Cpu.AssignedProcessorCount,
+		"cpu_thread_pool_available_worker_threads":                      serverResponse.Cpu.ThreadPoolAvailableWorkerThreads,
+		"cpu_thread_pool_available_completion_port_threads":             serverResponse.Cpu.ThreadPoolAvailableCompletionPortThreads,
+		"memory_allocated_in_mb":                                        serverResponse.Memory.AllocatedMemoryInMb,
+		"memory_installed_in_mb":                                        serverResponse.Memory.InstalledMemoryInMb,
+		"memory_physical_in_mb":                                         serverResponse.Memory.PhysicalMemoryInMb,
+		"memory_low_memory_severity":                                    mapMemorySeverity(serverResponse.Memory.LowMemorySeverity),
```

Comment:
Please keep the string here instead of mapping it to a cryptic number. A sting is much easier to discover during data-analysis especially by people that don't know this code.

Reply:
done

Reply:
Please mark as resolved.